app/models/cocktail.rb

```diff
@@ -14,9 +14,10 @@ def self.dashboard_five(_user_id)
       .limit(5)
   end
 
-  def self.sample_rated(sample_size)
+  def self.sample_rated(user_id, sample_size)
     joins(:ratings)
       .select('cocktails.*, ratings.stars as rating')
+      .where('user_id = ?', user_id)
       .sample(sample_size)
```

Comment:
@markcyen  Quick thought here: If we want only top-rated cocktails we could just take out the sample line and add something like this instead: 
```rb
.order('rating desc')
.limit(5)
```

Reply:
This could work. Makes it easier rather than passing in a second argument. Do you want me to make this change?

Reply:
Updated!